backend/mockingbird/src/main/scala/ru/tinkoff/tcb/mockingbird/model/GrpcStubView.scala

```diff
@@ -0,0 +1,55 @@
+package ru.tinkoff.tcb.mockingbird.model
+
+import java.time.Instant
+
+import derevo.circe.decoder
+import derevo.circe.encoder
+import derevo.derive
+import eu.timepit.refined.types.numeric.NonNegInt
+import eu.timepit.refined.types.string.NonEmptyString
+import io.circe.Json
+import io.circe.refined.*
+import io.scalaland.chimney.dsl.TransformationOps
+import sttp.tapir.codec.refined.*
+import sttp.tapir.derevo.schema
+
+import ru.tinkoff.tcb.predicatedsl.Keyword
+import ru.tinkoff.tcb.predicatedsl.json.JsonPredicate
+import ru.tinkoff.tcb.protocol.json.*
+import ru.tinkoff.tcb.protocol.schema.*
+import ru.tinkoff.tcb.utils.circe.optics.JsonOptic
+import ru.tinkoff.tcb.utils.id.SID
+
+@derive(encoder, decoder, schema)
+final case class GrpcStubView(
```

Comment:
nit: It looks like used only for response. So, maybe it would be better to move it to a new `response` package. What do you think?

Reply:
As there is no response package at all in project, there should be a separate refactoring for this purpose